mkdocs.yml

```diff
@@ -25,6 +25,22 @@ theme:
   features:
     - content.code.copy
     - navigation.indexes
+  palette:
```

Comment:
Key 'palette' is duplicated

Reply:
## before 

<img width="1024" alt="image" src="https://github.com/user-attachments/assets/ef6f2d6f-cfdd-42a1-99a0-fd4eeb13908d">

## after

<img width="1030" alt="image" src="https://github.com/user-attachments/assets/92c5ca29-30d2-48bb-b0c8-acf55e05b988">

I think we should not change colors

Reply:
resolved